en/docs/how-tos/preserve-override-headers.md

```diff
@@ -0,0 +1,69 @@
+# Preserve/Override Headers
+
+By default, some headers of the request and the response will be overridden in the API Microgateway. 
+If you have a requirement to preserve/override the request headers to be sent to the back-end or to preserve/override response headers to be received from the back-end.
+
+- [Preserve Request and Response Headers](#preserve-request-and-response-headers)
+- [Override Request and Response Headers](#override-request-and-response-headers)
+
+!!! note
+    This feature is available from WUM timestamp **1612370612556** of API Microgateway Toolkit 3.2.0.
```

Comment:
Lets add the date maps to the timestamp with in brackets as well,

Reply:
Added with https://github.com/wso2/docs-mg/pull/154/commits/c26d0e02a2449735ea783e442125efa1f08b67ef